<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocViewerDialog</class>
 <widget class="QDialog" name="DocViewerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Document Viewer - Qupil</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_print">
     <property name="text">
      <string>&amp;Print</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/qupil.qrc">
       <normaloff>:/gfx/document-print.svg</normaloff>:/gfx/document-print.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_exportToPdf">
     <property name="text">
      <string>Export to &amp;PDF</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/qupil.qrc">
       <normaloff>:/gfx/application-pdf.svg</normaloff>:/gfx/application-pdf.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/qupil.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DocViewerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
